readme.md

```diff
@@ -77,7 +77,7 @@ If you want to halt the boot when a property prefixed with `/` isn't found in th
 
 TL;DR: Define the enabling properties in the bootstrap properties (`bootstrap.yml`, `bootstrap.properties`, [etc.](https://cloud.spring.io/spring-cloud-static/spring-cloud.html#_the_bootstrap_application_context))(see [Unleashing the Magic](#there-are-3-ways-to-enable-this-lib-after-importing-it-in-your-pomxml-pick-yours)).
 
-Spring Cloud has a second application context named bootstrap that gets initialized before Spring Boot's normal application context. Since this lib uses an EnvironmentPostPrecessor to add the Parameter Store PropertySource, it gets triggered twice. Since you probably want it to be added on the bootstrap context, we highly recommend using the bootstrap properties to enable this lib. More details on this [Stack Overflow Thread](https://stackoverflow.com/questions/50935915/adding-a-conditional-external-propertysource-in-a-spring-boot-application).
+Spring Cloud has a second application context named bootstrap that gets initialized before Spring Boot's normal application context. Since this lib uses an EnvironmentPostPrecessor to add the Parameter Store PropertySource, it was triggered twice. Since you probably want it to be added on the bootstrap context, a flag was added to prevent any double initialization. We highly recommend using the bootstrap properties to enable this lib. More details on this [Stack Overflow Thread](https://stackoverflow.com/questions/50935915/adding-a-conditional-external-propertysource-in-a-spring-boot-application). If you still want it to run twice or if you are using [spring-boot-devtools](https://docs.spring.io/spring-boot/docs/current/reference/html/using-spring-boot.html#using-boot-devtools-restart) you can set the special property `awsParameterStorePropertySource.disableMultipleContextSupport` to `false`. If you are using Spring Cloud, this property _must_ be set in the bootstrap properties or it won't work.

```

Comment:
I'd do another paragraph for the "If you still want it to run [...]" section. Add a note that the parameter is optional too, I initially thought it was not with the last sentence.